integration-tests/ledger-signer.spec.ts

```diff
@@ -185,5 +188,88 @@ CONFIGS().forEach(({ lib, setup, rpc }) => {
         done();
       });
     })
+
+    describe('Verify that use of a ledger device works with bip32', () => {
```

Comment:
Consider replacing `bip32` with `the derivation curve BIP32_ED25519` for clarity